assets/js/WfHamburgerMenu.js

```diff
@@ -4,7 +4,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { UncontrolledDropdown, DropdownToggle, DropdownMenu, DropdownItem } from './components/Dropdown'
 import ImportModuleFromGitHub from './ImportModuleFromGitHub'
-import LocaleSwitcher from './i18n/LocaleSwitcher'
+import LocaleSwitcher, { enableLocaleSwitcher } from './i18n/LocaleSwitcher'
```

Comment:
Could we please call this `isLocaleSwitcherEnabled`?

(`enableLocaleSwitcher` starts with a verb, so the name implies `function enableLocaleSwitcher() { isLocaleSwitcherEnabled = true }`.)